### PR TITLE
Adding core weekly for October 2019, weeks 40 to 44

### PR DESCRIPTION
**Pull request by ttoine**
Needs a quick review, and a paragraph in News.

### CODE REVIEW

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 40 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   XXXX
+authors: [ PrestaShop ]
```
Comment (matks) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md:6]: You can modify the author 😉

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+- [78 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2019-09-30..2019-10-06), including [18 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2019-09-30..2019-10-06) on the core;
+- [61 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-09-30..2019-10-06) in the project repositories;
+- [71 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-09-30..2019-10-06), including [63 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-09-30..2019-10-06).
+        
+
+## Code changes in the 'develop' branch (for vXXXX)
```
Comment (matks) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]: Remove the `(for vXXXX)`

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 30th of September to Sunday 6th of October 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+todo
```
Comment (matks) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]: todo 😄 (or remove if there is no relevant content)

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+* [#15772](https://github.com/PrestaShop/PrestaShop/pull/15772): Tests - Fix global Vars in linkchecker, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+* [#15754](https://github.com/PrestaShop/PrestaShop/pull/15754): Tests - Change structure of file globals.js, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+* [#15735](https://github.com/PrestaShop/PrestaShop/pull/15735): Sanity Tests with puppeteer - Refacto of InstallShopTest, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+* [#15651](https://github.com/PrestaShop/PrestaShop/pull/15651): Add common FO page - Puppeteer. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+
+## Code changes in the '1.7.6.x' branch (for vXXXX)
```
Comment (matks) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]:
```suggestion
## Code changes in the '1.7.6.x' branch (for v1.7.6.2)
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+
+### Classic-rocket theme
+
+* [#100](https://github.com/PrestaShop/classic-rocket/pull/100): Change type for phone_mobile field. Thank you [@micka-fdz](https://github.com/micka-fdz)
+
+### productcomments
+
+* [#38](https://github.com/PrestaShop/productcomments/pull/38): Fix for product display in modal. Thank you [@Oksydan](https://github.com/Oksydan)
+
+### PrestaShop-1.6
```
Comment (matks) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]: We might need to improve this title in order for people not confuse it with main repository

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+
+* [#362](https://github.com/PrestaShop/docs/pull/362): Documentation : Add a language, by [@Progi1984](https://github.com/Progi1984)
+
+* [#361](https://github.com/PrestaShop/docs/pull/361): Add notes that db account with full privileges have to be used. Thank you [@Mirgen](https://github.com/Mirgen)
+
+* [#360](https://github.com/PrestaShop/docs/pull/360): Import severity and review process build articles into the doc, by [@matks](https://github.com/matks)
+
+* [#359](https://github.com/PrestaShop/docs/pull/359): Add Laragon as an WAMP environment. Thank you [@yannicka](https://github.com/yannicka)
+
+### blockreassurance
```
Comment (matks) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]:
```suggestion
### Block Reassurance module
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md (hunk 5 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]: yes. maybe adding legacy or something like that. BTW, maybe we should update the name of the repository or the repository title.

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md:6]: it's ok like that. after all, if we would count the pull request on Build too, I would be credited for those core-weekly.

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 40 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   XXXX
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]:
```suggestion
date:   2019-11-05 15:50:00
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md (hunk 2 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]:
```suggestion
## Code changes in the 'develop' branch
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+
+### Classic-rocket theme
+
+* [#100](https://github.com/PrestaShop/classic-rocket/pull/100): Change type for phone_mobile field. Thank you [@micka-fdz](https://github.com/micka-fdz)
+
+### productcomments
+
+* [#38](https://github.com/PrestaShop/productcomments/pull/38): Fix for product display in modal. Thank you [@Oksydan](https://github.com/Oksydan)
+
+### PrestaShop-1.6
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-40-2019.md]:
```suggestion
### PrestaShop-1.6 (legacy repository)
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-41-2019.md [hunk 9]
```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 41 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   XXXX
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-41-2019.md]:
```suggestion
date:   2019-11-05 15:50:00
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-41-2019.md [hunk 10]
```diff
@@ -0,0 +1,194 @@
...
+- [58 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2019-10-07..2019-10-13), including [16 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2019-10-07..2019-10-13) on the core;
+- [51 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-10-07..2019-10-13) in the project repositories;
+- [66 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-10-07..2019-10-13), including [55 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-10-07..2019-10-13).
+        
+
+## Code changes in the 'develop' branch (for vXXXX)
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-41-2019.md]:
```suggestion
## Code changes in the 'develop' branch
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-42-2019.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 42 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   XXXX
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-42-2019.md]:
```suggestion
date:   2019-11-05 15:50:00
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-42-2019.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+- [56 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2019-10-14..2019-10-20), including [17 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2019-10-14..2019-10-20) on the core;
+- [61 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-10-14..2019-10-20) in the project repositories;
+- [60 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-10-14..2019-10-20), including [50 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-10-14..2019-10-20).
+        
+
+## Code changes in the 'develop' branch (for vXXXX)
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-42-2019.md]:
```suggestion
## Code changes in the 'develop' branch
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-42-2019.md [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+
+* [#15905](https://github.com/PrestaShop/PrestaShop/pull/15905): Tests - Add module alias, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+* [#15865](https://github.com/PrestaShop/PrestaShop/pull/15865): Functional Tests - Add BO tests for customers, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+### Merge
+
+* [#15951](https://github.com/PrestaShop/PrestaShop/pull/15951): Merge modifications from branch 176 back to develop, by [@jolelievre](https://github.com/jolelievre)
+
+## Code changes in the '1.7.6.x' branch (for vXXXX)
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-42-2019.md]:
```suggestion
## Code changes in the '1.7.6.x' branch (for 1.7.6.2)
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-43-2019.md [hunk 14]
```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 43 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   XXXX
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-43-2019.md]:
```suggestion
date:   2019-11-05 15:50:00
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-43-2019.md [hunk 15]
```diff
@@ -0,0 +1,140 @@
...
+- [62 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2019-10-21..2019-10-27), including [11 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2019-10-21..2019-10-27) on the core;
+- [55 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-10-21..2019-10-27) in the project repositories;
+- [56 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-10-21..2019-10-27), including [33 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-10-21..2019-10-27).
+        
+
+## Code changes in the 'develop' branch (for vXXXX)
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-43-2019.md]:
```suggestion
## Code changes in the 'develop' branch
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-43-2019.md [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+
+* [#15988](https://github.com/PrestaShop/PrestaShop/pull/15988): Functional Tests - Add tests for categories. Thank you [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+
+* [#15980](https://github.com/PrestaShop/PrestaShop/pull/15980): Linkchecker update, by [@SimonGrn](https://github.com/SimonGrn)
+
+### Merge
+
+* [#16051](https://github.com/PrestaShop/PrestaShop/pull/16051): Merge of bug fixes from 1.7.6.x branch to develop, by [@matks](https://github.com/matks)
+
+## Code changes in the '1.7.6.x' branch (for vXXXX)
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-43-2019.md]:
```suggestion
## Code changes in the '1.7.6.x' branch (for 1.7.6.2)
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-44-2019.md [hunk 17]
```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 44 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   XXXX
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-44-2019.md]:
```suggestion
date:   2019-11-05 15:50:00
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-44-2019.md [hunk 18]
```diff
@@ -0,0 +1,153 @@
...
+- [61 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2019-10-28..2019-11-03), including [17 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2019-10-28..2019-11-03) on the core;
+- [58 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2019-10-28..2019-11-03) in the project repositories;
+- [52 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2019-10-28..2019-11-03), including [46 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2019-10-28..2019-11-03).
+        
+
+## Code changes in the 'develop' branch (for vXXXX)
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-44-2019.md]:
```suggestion
## Code changes in the 'develop' branch
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-44-2019.md [hunk 19]
```diff
@@ -0,0 +1,153 @@
...
+
+* [#16216](https://github.com/PrestaShop/PrestaShop/pull/16216): Tests - Add functional tests command to package.json, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+* [#16194](https://github.com/PrestaShop/PrestaShop/pull/16194): Tests - Refactoring puppeteer tests, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+* [#16156](https://github.com/PrestaShop/PrestaShop/pull/16156): Tests - Update name of catalog Menu link in BO, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+* [#16125](https://github.com/PrestaShop/PrestaShop/pull/16125): Functional Tests - Add BO tests for Brands, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+## Code changes in the '1.7.6.x' branch (for vXXXX)
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-44-2019.md]:
```suggestion
## Code changes in the '1.7.6.x' branch (for 1.7.6.2)
```

File: news/_posts/core-weekly/2019-11-05-coreweekly-week-41-2019.md [hunk 20]
```diff
@@ -0,0 +1,194 @@
...
+
+### Tests
+
+* [#15912](https://github.com/PrestaShop/PrestaShop/pull/15912): Fix missing currency in behat test, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+* [#15868](https://github.com/PrestaShop/PrestaShop/pull/15868): Update puppeteer Tests description on package.json, by [@boubkerbribri](https://github.com/boubkerbribri)
+
+* [#15848](https://github.com/PrestaShop/PrestaShop/pull/15848): Fix E2E dockerfile as npm is not anymore in the nodejs package, by [@PierreRambaud](https://github.com/PierreRambaud)
+
+## Code changes in the '1.7.6.x' branch (for vXXXX)
```
Comment (ttoine) [inline news/_posts/core-weekly/2019-11-05-coreweekly-week-41-2019.md]:
```suggestion
## Code changes in the '1.7.6.x' branch (for 1.7.6.2)
```

**Review by ttoine — COMMENTED** [review]
A few stuff to fix.

**Pull request merged**